packages/core/.storybook/public/preview.css

```diff
@@ -1,10 +1,10 @@
 @import '@cds/city/css/bundles/default.min.css';
```

Comment:
Storybook APIs changed a bit. This file was cleaned up to relfect the APIs. There is a css file for the preview/iframe and a new file for the css only to be applied to the manager UI. Any commonly shared styles are now in `common.css`